source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -129,6 +237,51 @@ This means that in order to have a VM connected to both ipv4 and ipv6 subnets,
 it needs to connect to 2 different networks (through 2 different logical ports),
 one with an IPv4, other with an IPv6 subnet defined on top.
 
+### Networking API entity provisioning
+
+Below you can find the steps required to have dynamic IPv6 addresses on the
+guest VMs, along with the available methods of provisioning it.
+
+1. create a logical network
+    - oVirt UI
+    - oVirt REST API
+    - manageIQ UI
+    - ovirt-provider-ovn REST API
+2. add a subnet on top
+    - oVirt UI - *requires changes*
+    - manageIQ UI
+    - ovirt-provider-ovn REST API
+3. provision a router
+    - ovirt-provider-ovn REST API
+4. attach the subnet to the router
+    - oVirt ovirt-provider-ovn REST API
```

Comment:
Please add example of the body request here

Reply:
Done